docs/docs/micromatch-wrapper.md

```diff
@@ -0,0 +1,52 @@
+#### Overview
+[Micromatch](https://github.com/micromatch/micromatch) support various matching features, such as Glob pattern, Advanced globbing.
+
+Micromatch is a nodejs application, and MBT is a go application. In order to enable MBT to use the functionalities provided by micromatch, from MBT v1.2.25, we create [micromath wrapper](https://github.com/SAP/cloud-mta-build-tool/tree/master/micromatch) to package it.
+
+#### Install
+You can install the Micromatch Wrapper using either of these methods below:

```

Comment:
You can install the micromatch wrapper using one of the methods below:

Reply:
OK